GameEngine/GameState.cs

```diff
@@ -62,7 +70,7 @@ public override int GetHashCode()
             {
                 hashCode = hashCode * -1521134295 + EqualityComparer<GameBoard>.Default.GetHashCode(Board);
                 hashCode = hashCode * -1521134295 + EqualityComparer<IDeck>.Default.GetHashCode(Deck);
-                hashCode = hashCode * -1521134295 + EqualityComparer<PlayerHand>.Default.GetHashCode(Hand);
+                hashCode = hashCode * -1521134295 + EqualityComparer<List<PlayerHand>>.Default.GetHashCode(Hands);
```

Comment:
We should lift the constant number into a named constant

Reply:
I suspect that the GetHashCode methods are generated by visual studio. 

- We should leave it alone because all of the other GetHashCode methods look this way.
- If I'm right this being a generated method then we can probably assume that having the constant present like this is the right choice vs having it stored in a constant field.
- Considering it's a mostly meaningless constant, putting it into a constant doesn't clarify anything.  Instead of a meaningless number literal we'll have a meaningless symbol literal (const int HashConstant = number).  Either way people are copying and pasting a thing.

Reply:
I agree with @dlinman